src/FE/src/app/pages/users-page/users-page.component.ts

```diff
@@ -0,0 +1,186 @@
+import {ChangeDetectorRef, Component, OnInit} from '@angular/core';
+import {NgClass, NgIf} from '@angular/common';
+import {MatButton, MatIconButton} from '@angular/material/button';
+import {
```

Comment:
tror denne kan være `import {MatTableDataSource, MatTableModule} from '@angular/material/table';` i stedet for alle disse individuelle